cmd/genyaml/gen_kubectl_yaml.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"gopkg.in/yaml.v2"
+	"k8s.io/kubernetes/cmd/genutils"
+	"k8s.io/kubernetes/pkg/kubectl/cmd"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+type cmdOption struct {
+	Name         string
+	Shorthand    string `yaml:",omitempty"`
+	DefaultValue string `yaml:"default_value,omitempty"`
+	Usage        string `yaml:",omitempty"`
+}
+
+type cmdDoc struct {
+	Name             string
+	Synopsis         string       `yaml:",omitempty"`
+	Description      string       `yaml:",omitempty"`
+	Options          []cmdOption  `yaml:",omitempty"`
+	InheritedOptions []cmdOption	`yaml:"inherited_options,omitempty"`
+	Example          string       `yaml:",omitempty"`
+	SeeAlso          []string     `yaml:"see_also,omitempty"`
+}
+
+func main() {
```

Comment:
Please take a look at https://github.com/kubernetes/kubernetes/blob/643cb7a1c7499df4e569f4f0fbd3b18c0c4e63ce/cmd/genkubedocs/gen_kube_docs.go

It would be much better to use that system than this system, if possible?